articles/multifactor-authentication/send-phone-message-hook-vonage.md

```diff
@@ -0,0 +1,129 @@
+---
+description: Configuring a Custom SMS Provider for MFA using Vonage
+topics:
+  - mfa
+  - sms
+  - custom-sms-provider
+contentType:
+  - how-to
+useCase:
+  - customize-mfa
+---
+# Configuring a Custom SMS Provider for MFA using Vonage
+
+This guide covers how to send multi-factor authentication text messages using the Vonage (previously Nexmo) SMS API.
+
+**Please Note:** The following steps will add text-message-based multifactor to the login flow for the tenant that you're working in. We **highly** recommend testing this setup on a [staging or development server](/dev-lifecycle/setting-up-env) before making the changes to your production login flow.
+
+## What is Vonage?
```

Comment:
IMO this should just be 

```
[Vonage](https://www.vonage.com/communications-apis/sms/) provides an SMS API which can be used by Auth0 to deliver multi-factor verification via text messages.
```

That says what it is; everything else is a weird sort of ad for Vonage.

And then in the next section, we should remove the screenshot of the Vonage UI. We try hard not to use lots of screenshots of our own UI, and we are trying to (going forward, and as we edit old docs) not screenshot other products' UI at all where possible. Maintenance burden and all.